tests/data/test1401

```diff
@@ -91,7 +91,7 @@ int main(int argc, char *argv[])
   curl_easy_setopt(hnd, CURLOPT_FTP_SKIP_PASV_IP, 1L);
 %endif
   curl_easy_setopt(hnd, CURLOPT_TCP_KEEPALIVE, 1L);
-  curl_easy_setopt(hnd, CURLOPT_PROTOCOLS_STR, "http,ftp,file");
+  curl_easy_setopt(hnd, CURLOPT_PROTOCOLS_STR, "http");
```

Comment:
This is the only test case verifying `CURLOPT_PROTOCOLS_STR` with `---libcurl`, shouldn't it use more than one protocol ?

We can just make the test require support for those protocols to be present.

Reply:
Yes, but how to check for 3 protocols in the same test? I didn't find a way to make the program "auto-adaptive" and still match the `<verify>` data.

Reply:
In the `<client>` section you list all features that are required:
~~~
<features>
http
ftp
file
</features>

Reply:
Thanks for the hint. I didn't know protocol names were features !

Reply:
Fixed now!